force_wfmanager/wfmanager_global_task.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import platform
+
+from pyface.tasks.action.api import SMenu, SToolBar, TaskAction
+from pyface.api import ImageResource
+from traits.api import List
+
+from force_wfmanager.wfmanager import TaskToggleGroupAccelerator
+
+from traits.api import Callable
+from envisage.ui.tasks.task_extension import TaskExtension
+from pyface.tasks.action.api import SchemaAddition
+
+log = logging.getLogger(__name__)
+
+IS_WINDOWS = (platform.system() == "Windows")
+
+
+class ExitAction(TaskAction):
+    """ A standard 'Quit' (Mac) or 'Exit' (Windows) menu action. """
+
+    accelerator = "Alt+F4" if IS_WINDOWS else "Ctrl+Q"
+    method = "setup_task.exit"
+
+    def _name_default(self):
+        return u"Exit" if IS_WINDOWS else u"Quit"
```

Comment:
I don't think this class is working properly - it's not appearing in the `File` drop down menu. Some simple unit tests might help to see whether the problem lies with the class definition, or where we are instantiating it in line 98.

Reply:
It shouldn't on Mac OS, it should appear in the main application menu (probably called "Python" when you typically run).  See https://doc.qt.io/qt-5/macos-issues.html#menu-bar

Reply:
Ah, I see - then I think the master branch had overridden this feature to provide an extra item in the `File` drop down:

<img width="329" alt="Screenshot 2020-04-20 at 16 48 57" src="https://user-images.githubusercontent.com/50372949/79771564-e14ae800-8326-11ea-9d46-1c107f230c65.png">

But this no longer appears in the current branch:

<img width="329" alt="Screenshot 2020-04-20 at 16 50 56" src="https://user-images.githubusercontent.com/50372949/79771753-1e16df00-8327-11ea-8e25-ee05186e3bae.png">

From the comments above, it seems that the current behaviour (exit only through `Python` from down) should be put in place, but addressed in a new PR.

Reply:
See #378